submissions/nadsatt/dom-api/app.js

```diff
@@ -0,0 +1,198 @@
+        rating: 8,
+        colors: ['rgb(216,80,28)', 'rgb(225,224,142)', 'rgb(118,96,84)', 'rgb(189,200,196)']
+    },
+    {
+        id: 1,
+        name: 'great catsby', 
+        description: 'Midwest native Nick Carraway arrives in 1922 New York in search of the American dream. Nick, a would-be writer, moves in next-door to millionaire Jay Gatsby and across the bay from his cousin Daisy and her philandering husband, Tom. Thus, Nick becomes drawn into the captivating world of the wealthy and -- as he bears witness to their illusions and deceits -- pens a tale of impossible love, dreams, and tragedy.',
+        rating: 6,
+        colors: ['rgb(7,229,169)', 'rgb(106, 115, 94)', 'rgb(155, 163, 146)', 'rgb(202, 181, 158)']
+    },
+    {
+        id: 2,
+        name: 'cat of the year', 
+        description: 'Tom Dobbs, the satirical host of a political talk show, decides to shake things up by running for president after making comments that he would be a better president than the leader who currently occupies the White House. His fans begin a grassroots campaign that catapults him into the Oval Office, but soon after moving in it is revealed a computer glitch mistakenly awarded him the presidency. Tom is left with a tough choice, stay the course or head back behind the microphone.',
+        rating: 2,
+        colors: ['rgb(207,163,179)', 'rgb(146,43,62)', 'rgb(23,69,111)', 'rgb(189,200,196)']
+    },
+    {
+        id: 3,
+        name: 'kitty potter', 
+        description: 'Harry Potter\'s third year at Hogwarts starts off badly when he learns deranged killer Sirius Black has escaped from Azkaban prison and is bent on murdering the teenage wizard. While Hermione\'s (Emma Watson) cat torments Ron\'s sickly rat, causing a rift among the trio, a swarm of nasty Dementors is sent to protect the school from Black. A mysterious new teacher helps Harry learn to defend himself, but what is his secret tie to Sirius Black?',
+        rating: 9,
+        colors: ['rgb(0,145,194)', 'rgb(0,26,57)', 'rgb(175, 171, 171)', 'rgb(116,153,156)']
+    },
+    {
+        id: 4,
+        name: 'new meov', 
+        description: 'High-school student Bella Swan, always a bit of a misfit, doesn\'t expect life to change much when she moves from sunny Arizona to rainy Washington state. Then she meets Edward Cullen, a handsome but mysterious teen whose eyes seem to peer directly into her soul. Edward is a vampire whose family does not drink blood, and Bella, far from being frightened, enters into a dangerous romance with her immortal soulmate.',
+        rating: 5,
+        colors: ['rgb(23,131,72)', 'rgb(93, 150, 18)', 'rgb(175, 171, 171)', 'rgb(128,126,44)']
+    },
+];
+
+function toggleSidebar(){
+    headerIcon.classList.toggle('icon-wrapper--active');
+    sidebarIcon.classList.toggle('icon-wrapper--active');
+    sidebar.classList.toggle('sidebar--active');
+}
+
+function selectSidebarItem(e){
```

Comment:
you can destructure your argument here
```suggestion
function selectSidebarItem({target}){
```